docs/ThreeLiner.md

```diff
@@ -0,0 +1,33 @@
+# Three Liner
+
+The `<pxb-three-liner>` accepts content that can grow and shrink within the context of a `<pxb-app-bar>`.
```

Comment:
```suggestion
The `<pxb-three-liner>` can display up to three lines of stylized text or other ng-content. It is most commonly used within the context of a `<pxb-app-bar>` where the text can grow / shrink as the App Bar is expanded and collapsed.
```

Reply:
Can you add a link to the app bar component docs as well?

Reply:
You got it! The link is added now.